pkgs/development/python-modules/ibmquantumexperience/default.nix

```diff
@@ -1,33 +0,0 @@
-{ lib
-, buildPythonPackage
-, fetchPypi
-, requests
-, requests_ntlm
-}:
-
-buildPythonPackage rec {
-  pname = "IBMQuantumExperience";
-  version = "2.0.4";
```

Comment:
why remove?

Reply:
``ibmquantumexperience`` package was deprecated a year ago in favor of ``qiskit-ibmq-provider``, and ``ibmquantumexperience`` is unused in-tree.

https://github.com/QISKit/qiskit-api-py